controller/test-files/root/list/ok_root.res.payload.golden.json

```diff
@@ -0,0 +1,140 @@
+{
+  "data": {
+    "id": "00000000-0000-0000-0000-000000000001",
+    "links": {
+      "self": "http:///api"
+    },
+    "relationships": {
```

Comment:
Your JSONAPI payload contains the members `id`, `links`, and `relationship` but it is missing an crucial part: the `type` of the resource (see http://jsonapi.org/format/#document-resource-objects).